csrc/host_ir/lower.cpp

```diff
@@ -592,6 +592,23 @@ std::vector<Expr*> HostIrLower::lowerToCollectiveBasedPipelinedGemmComm(
       get_current_stream, allocate_tva_allgathered, allocate_tv_out, for_loop};
 }
 
+bool HostIrLower::isLowerableAsStandaloneHostOp(Expr* expr) {
+  return isResharding(expr);
```

Comment:
Are you expecting this to change in the future to something other than `return isResharding(expr)`? I'm just wondering, why not just use `isResharding` like before?

Reply:
yes, this will change from https://github.com/NVIDIA/Fuser/pull/4146 on